partials/next-previous-tabs.php

```diff
@@ -23,27 +23,47 @@
     if (!empty($next_page)) {
         echo 'has-next-page';
     } ?>">
-
+      <ul class="gem-c-pagination__list">
```

Comment:
Can we discuss this. It looks good im just curious why its not using the govuk classes.

Reply:
There are no equivalent govuk classes for pagination - so they are not part of the GDS design system.  They are, however, used in gov publishing, so I have copied their CSS and kept their class names.

Reply:
Pagination doesn't have govuk classes - I have copied the pagination from the government publishing styles and not changed the class names - so it is easier to know where they come from.